resources/views/emails/community_service_requested.blade.php

```diff
@@ -0,0 +1,16 @@
+@component('mail::message')
+    <h1>Kedves {{ $recipient->name }}!</h1>
+    <p>
+        {{ $requester->name }} új közösségi tevékenységet hozott létre a következő leírással: <br>
+        "{{ $description }}"<br>
+        Kérlek fogadd el vagy utasítsd el a tevékenységet a következő linken a táblázatban: <br>
+        @component('mail::button', ['url'=> route('community_service')])
```

Comment:
You can forward this to the search route with the chosen user

Reply:
I don't know how I can do that.